core-feature-pack/src/main/resources/content/bin/common.ps1

```diff
@@ -167,6 +167,74 @@ Param(
   return $PROG_ARGS
 }
 
+# JBossws-cxf utilites require additional input args to be passed to the pgm
+Function Jbossws-Get-Java-Arguments {
```

Comment:
Besides the addition of the `[string]$javaOpts` what's the difference between this and `Get-Java-Arguments`?

Reply:
There are 2 input arg differences [string]$scriptName and [string]$javaOpts.   JAVA_OPTS is handled differently in Jbossws-Get-Java-Arguments.